modules/etcd-cert-alerts-metrics-signer.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * security/certificate_types_descriptions/etcd-certificates.adoc
+
+:_mod-docs-content-type: CONCEPT    
+[id="etcd-cert-alerts-metrics-signer_{context}"]
+== etcd certificate rotation alerts and metrics signer certificates
```

Comment:
```suggestion
= etcd certificate rotation alerts and metrics signer certificates
```
The first heading in a module is always an H1. Adjust the leveloffset in the assembly, if necessary.